packages/core/test/financial-templates/common/financial-product-libraries/expiring-multiparty-libraries/PreExpirationIdentifierTransformationFinancialProductLibrary.js

```diff
@@ -1,98 +1,110 @@
+const hre = require("hardhat");
+const { runDefaultFixture } = require("@uma/common");
+const { getContract } = hre;
 const { didContractThrow } = require("@uma/common");
 const { assert } = require("chai");
 
 // Tested Contract
-const PreExpirationIdentifierTransformationFinancialProductLibrary = artifacts.require(
+const PreExpirationIdentifierTransformationFinancialProductLibrary = getContract(
   "PreExpirationIdentifierTransformationFinancialProductLibrary"
 );
 
 // Helper contracts
-const Timer = artifacts.require("Timer");
-const ExpiringMultiPartyMock = artifacts.require("ExpiringMultiPartyMock");
+const Timer = getContract("Timer");
+const ExpiringMultiPartyMock = getContract("ExpiringMultiPartyMock");
 
 const { toWei, hexToUtf8, utf8ToHex } = web3.utils;
 const priceFeedIdentifier = utf8ToHex("TEST_IDENTIFIER");
 const transformedPriceFeedIdentifier = utf8ToHex("TEST_IDENTIFIER_TRANSFORMED");
 const collateralizationRatio = toWei("1.2");
 
-contract("PreExpirationIdentifierTransformationFinancialProductLibrary", function () {
+describe("PreExpirationIdentifierTransformationFinancialProductLibrary", function () {
   let identifierTransformationFPL;
   let expiringMultiParty;
   let timer;
   let expirationTime;
+  let accounts;
 
-  beforeEach(async () => {
+  before(async () => {
+    await runDefaultFixture(hre);
+    accounts = await hre.web3.eth.getAccounts();
     timer = await Timer.deployed();
 
-    expirationTime = (await timer.getCurrentTime()) + 100; // use 100 seconds in the future as the expiration time.
-    identifierTransformationFPL = await PreExpirationIdentifierTransformationFinancialProductLibrary.new();
+    expirationTime = (await timer.methods.getCurrentTime().call()) + 100; // use 100 seconds in the future as the expiration time.
+    identifierTransformationFPL = await PreExpirationIdentifierTransformationFinancialProductLibrary.new().send({
```

Comment:
if you _did not_ put the `{ from: accounts[0] }` here would it default to using this account or what would have happened? does it error out in that case?

Reply:
It does error, yeah.